docs/docs/contribute/code/build-shards.md

```diff
@@ -35,29 +35,26 @@ Go to the [Shard’s repository on Github](https://github.com/fragcolor-xyz/shar
 The repository is now cloned to your computer! Take note of where it has been cloned to, as we will be using it in the following step…
 
 
-## Navigating to the Shards Repository ##
+## Navigating to the Shards Repository
 
-Launch the MinGW terminal (ming64.exe). The .exe file can be found in the msys64 folder, usually located in your C drive.
+Open up Git Bash.
 
-![Find and select the ming64.exe](assets/location-mingw.png)
+??? Tip
+    You can use the Windows Search Bar to find the Git Bash application.
 
 Navigate to where your Shards repository is located using the command `cd $(cygpath -u '(X)')`, where (X) is the directory of your folder.
 
-```
-cd $(cygpath -u '(X)')
-```
-
 ??? "cd"
-	The `cd` command is used for navigating to different directories within the MinGW terminal.
+    The `cd` command is used for navigating to different directories within the terminal.
 
 ??? "cygpath"
-    The MinGW terminal requires you to convert the backslashes(\\) in your directory address to forward slashes(/). You will also have to remove the colon(:) after the drive alphabet. 
+    The Git Bash terminal requires you to convert the backslashes(\\) in your directory address to forward slashes(/). You will also have to remove the colon(:) after the drive alphabet. 
     
     This is the Unix form of an address, and can be done automatically by using the cygpath program.
 
     e.g. `cygpath -u 'C:\Fragcolor\Shards'` will output `/c/Fragcolor/Shards`
 
-If your Shards repository is located at `C:\Fragcolor\Shards`, the command used in the MinGW terminal would be `cd $(cygpath -u 'C:\Fragcolor\Shards')`.
+If your Shards repository is located at `C:\Fragcolor\Shards`, the command used in the terminal would be `cd $(cygpath -u 'C:\Fragcolor\Shards')`.
```

Comment:
We'll have to remove the mention of Fragcolor here too.

Reply:
I'll simplify to just `C:\Shards` then